### PR TITLE
Unitest update users

### PR DESCRIPTION
**Pull request by mvn-nale-dn**
## Description
Unitest Update User
## Reference Requirements
- Task ID: 25

## Reference Isuees
#{issue_id}

## Migrations
YES

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: tests/Browser/Pages/Backend/ShowListBookTest.php [hunk 1]
```diff
@@ -31,7 +31,8 @@ public function setUp()
     /**
      * A Dusk test show list book.
      *
-     * @return void
+     * @re
+     * turn void
      */
```
Comment (at-haphan) [inline tests/Browser/Pages/Backend/ShowListBookTest.php]: sửa lại chữ return

File: tests/Browser/Pages/Backend/UpdateUserTest.php [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+      /**
+     * Test Url Admin Edit Users.
+     *
+     * @return void
+     */
+    public function  testUrlEditUsers()
+    {
+        $user = User::find(1);
+        // dd($user->id);
+        $this->browse(function (Browser $browser) use ($user) {
```
Comment (at-haphan) [inline tests/Browser/Pages/Backend/UpdateUserTest.php]: xóa dòng lệnh k dùng đi

File: tests/Browser/Pages/Backend/UpdateUserTest.php [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+                    ->press('Submit')
+                    ->assertSee('Successfully updated user!')
+                    ->assertPathIs('/admin/users');
+        });
+        $this->assertDatabaseHas('users', [
+                        'name' => $user->name]);
+    }
+     /**
+    
+    /**
```
Comment (at-haphan) [inline tests/Browser/Pages/Backend/UpdateUserTest.php]: xóa dấu comment dư đi

File: tests/Browser/Pages/Backend/UpdateUserTest.php [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+            $browser->visit('/admin/users/'.$user->id.'/edit')
+                    ->resize(900,1000)
+                    ->assertSee('Update users')
+                    ->type('name',$user->name)
+                    ->press('Submit')
+                    ->assertSee('Successfully updated user!')
+                    ->assertPathIs('/admin/users');
+        });
+        $this->assertDatabaseHas('users', [
+                        'name' => $user->name]);
```
Comment (at-haphan) [inline tests/Browser/Pages/Backend/UpdateUserTest.php]: thêm test database

**Comment by at-haphan** [comment]
LGTM

**Comment by at-huydq** [comment]
LGTM

**Pull request merged**